rq/serializers.py

```diff
@@ -0,0 +1,42 @@
+import pickle
+
+from .exceptions import DeserializationError
+
+
+class MySerializer(object):
+    """MySerializer Class that uses the user specified serializer module."""
+    def __init__(self, serializer):
+        self.serializer = serializer
+
+    def dumps(self, data):
+        try:
+            serialized_data = self.serializer.dumps(data)
+        except Exception as e:
+            raise DeserializationError("Unserializable return value", data, e)
+
+        return serialized_data
+
+    def loads(self, serialized_string):
+        try:
+            obj = self.serializer.loads(serialized_string)
+        except Exception as e:
+            raise DeserializationError('Could not deserialize', serialized_string, e)
+
+        return obj
+
+
+def resolve_serializer(serializer):
```

Comment:
`resolve_serializer` must also be able to import a serializer from a string, for example `path.to.serializer`, similar to how you can specify a string to [import job class](https://github.com/rq/rq/blob/master/rq/queue.py#L73).

This is needed when we run `rq worker --serializer=path.to.serializer`